packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -789,6 +793,10 @@ describe("<DateRangeInput>", () => {
                 changeInputText(endInput, "");
             });
 
+            afterEach(() => {
+                rootInstance.reset();
```

Comment:
i wonder if this can be done with enzyme `wrapper.setProps(DRI.defaultProps)` and relying on `componentWillReceiveProps` instead of introducing a test-only instance method?